source/CoordinateConversion/ArcMapAddinCoordinateConversion/ViewModels/CollectTabViewModel.cs

```diff
@@ -693,17 +693,25 @@ public override void OnDisplayCoordinateTypeChanged(CoordinateConversionLibrary.
         public override void OnMapPointSelection(object obj)
         {
             var mapPoint = obj as IPoint;
-            mapPoint.Project(ArcMap.Document.FocusMap.SpatialReference);
-            ITopologicalOperator topoOp = (ITopologicalOperator)mapPoint;
-            IGeometry p2Bufferd = topoOp.Buffer(20000);
+            double dblSrchDis = 0;
             AddInPoint closestPoint = null;
             Double distance = 0;
 
+            var mxdoc = ArcMap.Application.Document as IMxDocument;
+            var av = mxdoc.FocusMap as IActiveView;

```

Comment:
Here is an example of a code section where null checks would be good to add. Some government programs run these checks against our code and this section will fail - these were issues #444 and #436 from a past run against the code.